src/guide/components/events.md

```diff
@@ -287,3 +288,17 @@ export default {
 ```
 
 </div>
+
+## Events as Props {#events-props}
+
+You may also declare and pass `events` as `props`, by prefixing the event name with `on` + Capitalized name.
+
+:::info
```

Comment:
**suggestion**: we're trying to reduce the number of highlighted blocks. Maybe we can merge those two in one block?

Reply:
the `info` could be remove the warning is something that should stay since is not something we don't want to recommend :D https://github.com/vuejs/core/pull/8025

Reply:
@pikax let's remove the `info` then :)